src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -117,7 +117,7 @@ export function getUiSettingDefaults() {
       name: i18n.translate('kbn.advancedSettings.dateFormatTitle', {
         defaultMessage: 'Date format',
       }),
-      value: 'MMMM Do YYYY, HH:mm:ss.SSS',
```

Comment:
Did this have to change for some reason?

Reply:
That is the format that design recommended so it fits in the UI better. Spelling out the month wastes a lot of space